.pre-commit-config.yaml

```diff
@@ -37,6 +36,11 @@ repos:
     - from __future__ import print_function
     - --remove-import
     - from __future__ import unicode_literals
+- repo: https://github.com/asottile/pyupgrade
+  rev: v3.12.0
+  hooks:
+  - id: pyupgrade
+    args: [--py37-plus]  # based on setup.cfg classifiers
```

Comment:
I'm not comfortable with this change as syntax between Python 3.5/3.6/3.7 is slightly different.
Given that this PR aims to drop Python 2 support (as specified in the package setup) I would not impose the fact that at least python 3.7 needs to be used.

Is the proposal of this PR to drop all the Python version that are in EOL (https://devguide.python.org/versions/)?
If so what is the rationale?
If supporting a "deprecated" version has no cost, what is the advantage of removing the support by introducing a different syntax?